src/main.rs

```diff
@@ -103,8 +103,12 @@ fn get_kernel_metrics(metrics: &mut HashMap<String, MetricValue>) {
         data
     };
     metrics.insert("max.res.size".into(), data.ru_maxrss.into());
-    metrics.insert("user.time".into(), ms_from_timeval(data.ru_utime).into());
-    metrics.insert("system.time".into(), ms_from_timeval(data.ru_stime).into());
+    let utime = ms_from_timeval(data.ru_utime);
+    let stime = ms_from_timeval(data.ru_stime);
+    metrics.insert("user.time".into(), utime.into());
+    metrics.insert("system.time".into(), stime.into());
+    let pct = (utime + stime) * 100.0 / wall_time;
+    metrics.insert("cpu.pct.wall.time".into(), pct.into());
```

Comment:
Why is this metric needed? Percentages can be calculated dynamically.

Reply:
I'm the one who asked for this metric to be included explicitly 🙂